Do not create a tar.gz archive of the source code.
https://sourceforge.net/p/ggt/bugs/30/
--- SConstruct.orig	2023-03-10 01:05:19.000000000 -0600
+++ SConstruct	2023-03-10 01:05:19.000000000 -0600
@@ -775,7 +775,6 @@
 
    pkg.build()
    installed_targets += pkg.getInstalledTargets()
-   MakeSourceDist(pkg, env)
 
 # Build everything by default
 Default('.')
